clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -4069,6 +4075,7 @@ void CXXNameMangler::mangleExpression(const Expr *E, unsigned Arity,
   case Expr::OMPArrayShapingExprClass:
   case Expr::OMPIteratorExprClass:
   case Expr::CXXInheritedCtorInitExprClass:
+  case Expr::UniqueStableNameExprClass:
```

Comment:
Hmmm, aren't there situations in which this expression would need to be mangled (like when used in a decltype expression as a templated function return type), or are those handled automatically?

Reply:
I have no idea, maybe?!  This is probably worth a TODO and a 'to test' then. At the moment, the idea was to get this to compile.